app/src/main/java/com/android/sample/model/recipe/DefaultRecipeRepository.kt

```diff
@@ -100,16 +101,6 @@ class DefaultRecipeRepository(
     networkRepository.searchByCategory(category, onSuccess, onFailure, limit)
   }
 
-  /**
```

Comment:
**Removing `listCategories`:**  
   Nice call removing `listCategories` if it’s no longer needed—keeps the repository lean and avoids redundant code. Just double-check that no other parts of the app rely on it.